js/ui/grid_core/ui.grid_core.columns_resizing_reordering.js

```diff
@@ -733,7 +733,7 @@ const ColumnsResizerViewController = modules.ViewController.inherit({
             that._trackerView.show();
             const scrollable = that.component.getScrollable();
 
-            if(scrollable && that.option('rtlEnabled') && that._isRtlMode()) {
+            if(scrollable && that.option('rtlEnabled') && that._isRtlParentStyle()) {
```

Comment:
isRtlParentStyle will check for this.option('rtlEnabled') so check rtlEnabled here is unnecessary